library/kotlin/src/io/envoyproxy/envoymobile/BUILD

```diff
@@ -36,7 +36,11 @@ kt_android_library(
 envoy_mobile_kt_library(
     name = "envoy_interfaces_lib",
     srcs = glob([
-        "EnvoyClient.kt",
+        "Counter.kt",
+        "Element.kt",
```

Comment:
I think it'd make sense to group these into a `stats` subdirectory, similarly to `grpc` and `filters`

Reply:
Sounds good to me.